activestorage/test/dummy/config/puma.rb

```diff
@@ -28,7 +28,11 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+if ENV["PIDFILE"]
+  pidfile ENV["PIDFILE"]
+else
+  pidfile tmp/pids/server.pid" if ENV.fetch("RAILS_ENV", "development") == "development"
```

Comment:
@hschne There is missing `"`

```diff
- pidfile tmp/pids/server.pid" if ENV.fetch("RAILS_ENV", "development") == "development"
+ pidfile "tmp/pids/server.pid" if ENV.fetch("RAILS_ENV", "development") == "development"
```

Reply:
Also whad do you think about simplifying the if-condition to reflect what we exactly say in the changelog: " Disable `pidfile` generation in production environment."

```
unless ENV["RAILS_ENV"] == "production"
```

Would that work?

Reply:
Looks like `pidfile` generation in `railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt` was updated further by https://github.com/rails/rails/pull/50669.

Wondering whether we should sync 

- `actionmailbox/test/dummy/config/puma.rb`
- `actiontext/test/dummy/config/puma.rb`
- `activestorage/test/dummy/config/puma.rb`

files with `railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt`